src/oslo_aero_coag.F90

```diff
@@ -659,15 +659,15 @@ subroutine clcoag(q, pmid, pdel, temperature, cldnum, cldfrc, delt_inverse, ncol
        end do ! i
     end do    ! k
 
-    !UPDATE THE TRACERS AND DO DIAGNOSTICS
+    ! UPDATE THE TRACERS AND DO DIAGNOSTICS
     do iCoagulator = 1, numberOfCoagulatingModes
        do ispecie = 1, getNumberOfTracersInMode(coagulatingMode(iCoagulator))
           l_index_donor = getTracerIndex(coagulatingMode(iCoagulator) , ispecie ,.true.)
 
           !so4_a1 is a process mode (condensate), but is still lost in coagulation
-          if( .NOT. is_process_mode(l_index_donor, .true.)          &
-              .OR. ( (l_index_donor.eq.chemistryIndex(l_so4_a1)) .AND. coagulatingMode(iCoagulator) &
-              .eq. MODE_IDX_OMBC_INTMIX_COAT_AIT) ) then
+          if( .NOT. is_process_mode(l_index_donor, .true.) .or. &
+               ( (l_index_donor == chemistryIndex(l_so4_a1)) .and. &
+                  coagulatingMode(iCoagulator) == MODE_IDX_OMBC_INTMIX_COAT_AIT) ) then
```

Comment:
There are parentheses around the first equality check but not the second one. Something to fix someday?